less/home-page.less

```diff
@@ -12,18 +12,49 @@
       padding:0 15px;
     }
   
+    @keyframes introimg {
+      0% {left: -1000px;}
+      100% {left: 0px;}
+    }
+
+    img {
+    position: relative;
+    animation-name: introimg;
+    animation-duration: 4s;
+    }
+
+    @media @phone {
+      img {
+        height: 13rem;
+        width: 100%;
+    }
```

Comment:
this is totally fine to do your media query with vanilla css syntax, but if you want to utilze less nesting, you can just put the media query inside of your img selector like this : 

@media @phone {
     height : 13rem;
     width : 100%;
}